src/Database/Database.php

```diff
@@ -759,7 +760,8 @@ public function createAttribute(string $collection, string $id, string $type, in
             case self::VAR_FLOAT:
             case self::VAR_BOOLEAN:
             case self::VAR_DATETIME:
-                break;
+            case self::VAR_ID:
+            break;
```

Comment:
Let's fix the indentation here:

```suggestion
                break;
```